include/deal.II/numerics/vector_tools.h

```diff
@@ -317,61 +319,238 @@ namespace VectorTools
 {
   /**
    * Denote which norm/integral is to be computed by the
-   * integrate_difference() function of this namespace. The following
-   * possibilities are implemented:
+   * integrate_difference() function on each cell and compute_global_error()
+   * for the whole domain.
+   * Let $f:\Omega \rightarrow \mathbb{R}^c$ be a finite element function
+   * with $c$ components where component $c$ is denoted by $f_c$ and $\hat{f}$
+   * be the reference function (the @p fe_function and @p exact_solution
+   * arguments to integrate_difference()). Let $e_c = \hat{f}_c - f_c$
+   * be the difference or error between the two. Further,
+   * let  $w:\Omega \rightarrow \mathbb{R}^c$ be the @p weight function of integrate_difference(), which is
+   * assumed to be equal to one if not supplied. Finally, let $p$ be the
+   * @p exponent argument (for $L_p$-norms).
+   *
+   * In the following,we denote by $E_K$ the local error computed by
+   * integrate_difference() on cell $K$, whereas $E$ is the global error
+   * computed by compute_global_error(). Note that integrals are
+   * approximated by quadrature in the usual way:
+   * @f[
+   * \int_A f(x) dx \approx \sum_q f(q) \omega_q.
```

Comment:
it should be `f(x_q)` where `x_q` is the q-th quadrature point mapped from the reference cell to the domain A.